pandas/core/generic.py

```diff
@@ -5291,47 +5288,162 @@ def replace(self, to_replace=None, value=None, inplace=False, limit=None,
                'piecewise_polynomial' in scipy 0.18; backwards-compatible with
                scipy < 0.18
 
-        axis : {0, 1}, default 0
-            * 0: fill column-by-column
-            * 1: fill row-by-row
-        limit : int, default None.
-            Maximum number of consecutive NaNs to fill. Must be greater than 0.
+        axis : {0 or 'index', 1 or 'columns', None}, default None
+            Axis to interpolate along.
+        limit : int, optional
+            Maximum number of consecutive NaNs to fill. Must be greater than
+            0.
+        inplace : bool, default False
+            Update the data in place if possible.
         limit_direction : {'forward', 'backward', 'both'}, default 'forward'
-        limit_area : {'inside', 'outside'}, default None
-            * None: (default) no fill restriction
-            * 'inside' Only fill NaNs surrounded by valid values (interpolate).
-            * 'outside' Only fill NaNs outside valid values (extrapolate).
-            .. versionadded:: 0.21.0
-
             If limit is specified, consecutive NaNs will be filled in this
             direction.
-        inplace : bool, default False
-            Update the NDFrame in place if possible.
+        limit_area : {`None`, 'inside', 'outside'}
```

Comment:
Add ", default None" to the end here and remove the comment about it being the default below